packages/design-system-docs/src/pages/components/Alert/_Alert.docs.scss

```diff
@@ -191,16 +191,32 @@ Style guide: components.alert.guidance
 /*
 Google Analytics
 
-On application where the page has `utag` loaded, analytics event tracking is enabled by default. The data goes to Tealium which allows it to route to Google Analytics or the currently approved data analytics tools.
-- To disable the automatic event tracking, pass this prop to the component
+**Analytics event tracking is disabled by default.**
```

Comment:
I updated the formatting of the content to use headers  making the sections easier to scan